portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,183 @@
+   
+    <script src="https://use.fontawesome.com/d1341f9b7a.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <body>
+
+    <nav >
+        <ul>
+      <li><a href="#contact">contact</a></li>
+     <li><a href="#projects">projects</a></li>
+        <li><a href="#experience ">experience</a></li>
+        <li class="empty"></li>
+        <li><a href="https://twitter.com/HumanCyyborg" target="_blank">twitter</a></li>
+       <li><a href="https://www.linkedin.com/in/danieldawsonkalu/" target="_blank">Linkedin</a></li>
+        
+    </ul>
+
+    </nav>
+
+    <header id="experience">
+       <div class ="container">
+         <div class ="row"> 
+<!-- left row -->
+             <div class="col-sm">
+        <h1>hi ,i'm<br/> daniel KALU</h1>
+            <ul>
+        <li>_Software developer & Ux designer</li>
+        <li>_Studying computer science at Minerva schools</li>
+        <li>_Cofounder of non profit projects and organisation</li>
+        <li>_content creator on Youtube</li>
+        <li>_speaking at various conferences for education</li>
+       </ul>
+    <a href="" class="button">add me on Linkedin</a>
+
+    </div>  
+    <!-- right row  -->
+      <div class="col-sm"> 
+    <figure>
+    <img src = "/image/daniel.jpg" class="intro-img" alt="daniel KALU" />
+    <figcaption>photo by sk</figcaption>
+    </figure>
+            </div>
+         </div>
+     </div>
+    
+    </header>
+
+    <section>
+        <!-- This section is for my bio -->
+    <ul class="skill-list">
+        <li>01_graphic design</li>
+        <li> 02_user experience</li>
+        <li>03_user interface design</li>
+        <li>04_design thinking</li>
+        <li>05_user research</li> 
+    </ul>
+    </section>
+    <!-- the first project section -->
+<section class="project blue" id="projects">
+     <div class ="container">
+         <div class ="row"> 
+<!-- left row -->
+             <div class="col-sm">
+   <figure>
+    <img src = "/image/lion.jpg" alt="daniel KALU" />
+    <figcaption class="right-alignment"> 
+        photo by sk
+    </figcaption>
+    </figure>
+             </div>
+
+
+<!-- right column -->
+      <div class="col-sm"> 
+    <h2>
+        Branding student<br/> for the good of technology
+    </h2>
+<p>
+    for female cosmetics
+Lorem ipsum dolor amet sartorial quinoa 3 wolf moon,
+ tilde neutra succulents vegan kitsch church-key vaporware. 
+ Pork belly thundercats iPhone tilde 90’s ugh. Narwhal post-ironic
+  VHS affogato subway tile. Everyday carry copper mug art party, 
+  blue bottle normcore banh mi gastropub lomo fingerstache four 
+  dollar toast single-origin coffee hell of 3 wolf moon palo santo kinfolk.
```

Comment:
Is it intentional to keep this text placeholder here and below?

Reply:
Yes, I will come back to it later and and add real content,just using it as place holder